src/Tribe/Views/V2/iCalendar/iCalendar_Handler.php

```diff
@@ -35,9 +36,10 @@ class iCalendar_Handler extends \tad_DI52_ServiceProvider {
 	protected $default_feeds = [
 		Google_Calendar::class,
 		iCal::class,
-		iCalendar_Export::class,
 		Outlook_365_Export::class,
 		Outlook_Live_Export::class,
```

Comment:
It's not a big deal - and it may not have been clear, but the "_Export" class(es) were originally just for the download, specifically to help differentiate between iCal/iCalendar (export vs. subscribe)

Reply:
Good to know. I renamed them.